www/src/data/sidebars/doc-links.yaml

```diff
@@ -14,6 +14,27 @@
       link: /starters/
     - title: Awesome Gatsby Resources
       link: /docs/awesome-gatsby-resources/
+    - title: Recipes
```

Comment:
It looks like there's two links in the sidebar to recipes (the other  on Lines 9-10). Is that intentional?

![image](https://user-images.githubusercontent.com/21114044/70819823-5f551c00-1d94-11ea-87e8-b4d20cb6bee7.png)

Reply:
Nope, probably a merge issue! The first standalone link can go away. I'll make that change shortly. Good catch! 👍